### DISCUSSION TITLE
6254BD Support

### DISCUSSION THREAD

**Discussion by drom** [OpenHantek/OpenHantek6022#398] [Q&A] [answered]
Great project!

I just got my Hantek 6254BD. Can't find any information if OpenHantek6022 supports my device?

**Comment by Ho-Ro** [comment, marked as answer]
>  Can't find any information if OpenHantek6022 supports my device?

If it is not mentioned, the OH6022 does not support it :)
Only devices based on the bare Cypress EzUSB (without additional FPGA) are supported. Typically these devices are announced with a sampling rate of 48 MS/s (even if the maximum usable rate is 24 or 30 MS/s).

**Reply by drom** [reply]
Thank you will keep searching for other OSS projects supporting my device ;)

**Comment by jimsy3** [comment]
where are the supported devices mentioned?

**Comment by Ho-Ro** [comment]
> where are the supported devices mentioned?

In the [README.md](https://github.com/OpenHantek/OpenHantek6022#openhantek6022), it mentions some rebadged brands:
> OpenHantek6022 is a free software for Hantek DSO6022 USB digital signal oscilloscopes that is actively developed on [github.com/OpenHantek/OpenHantek6022](https://github.com/OpenHantek/OpenHantek6022) - but only for Hantek 6022BE/BL and compatible scopes (Voltcraft, Darkwire, Protek, Acetech, etc.).

Technical requirements for support are mentioned in my 1st reply above:
> Only devices based on the bare Cypress EzUSB (without additional FPGA) are supported. Typically these devices are announced with a sampling rate of 48 MS/s (even if the maximum usable rate is 24 or 30 MS/s).

The 6254BD clearly doesn't fall into this category due to its [specs](https://www.hantek.eu/product/hantek-6254bd/):
> 4CH​ oscilloscope & function/Arb. waveform generator. ... It has 4 independent analog channels, 1GSa/s real-time sampling rate ...
